server/routerlicious/packages/services/src/redisThrottleStorageManager.ts

```diff
@@ -31,6 +32,7 @@ export class RedisThrottleStorageManager implements IThrottleStorageManager {
 
         client.on("error", (error) => {
             winston.error("Throttle Manager Redis Error:", error);
+            Lumberjack.error("Throttle Manager Redis Error:", { [CommonProperties.telemetryGroupName]: "throttling" }, error);
```

Comment:
you can remove the colon, and have the message be "Throttle Manager Redis Error"